src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -115,6 +115,9 @@ def get(self, request, organization):
         expand = request.GET.getlist("expand", [])
         collapse = request.GET.getlist("collapse", [])
         has_inbox = features.has("organizations:inbox", organization, actor=request.user)
+        has_workflow_owners = features.has(
+            "organizations:workflow-owners", organization, actor=request.user
```

Comment:
do we need to add this to the features `default_manager`?

Reply:
Yeah :)
https://github.com/getsentry/sentry/pull/22241/

Reply:
oh my b didn't see that

Reply:
I added it after your comment 😎

Reply:
oh haha i take it back then 😎